datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -76,35 +76,43 @@ use super::AggregateExec;
 
 /// This encapsulates the spilling state
 struct SpillState {
-    /// If data has previously been spilled, the locations of the
-    /// spill files (in Arrow IPC format)
-    spills: Vec<RefCountedTempFile>,
-
+    /* ======================================================================== */
+    // PROPERTIES:
+    // These fields are initialized at the start and remain constant throughout
+    // the execution.
+    /* ======================================================================== */
     /// Sorting expression for spilling batches
     spill_expr: Vec<PhysicalSortExpr>,
 
     /// Schema for spilling batches
     spill_schema: SchemaRef,
 
-    /// true when streaming merge is in progress
-    is_stream_merging: bool,
-
     /// aggregate_arguments for merging spilled data
     merging_aggregate_arguments: Vec<Vec<Arc<dyn PhysicalExpr>>>,
 
     /// GROUP BY expressions for merging spilled data
     merging_group_by: PhysicalGroupBy,
+
+    /* ======================================================================== */
```

Comment:
I dont remember we use this kind of format. I would probably stick to a single format across the project. Or we fill the format can be improved lets do it everywhere and add the notes to the contributing guide

Reply:
I can't find datafusion style guide, so I'm assuming the practice is following the existing style
I searched the code base and found comment separator `// ========`, I'll update to keep consistent with this format